hack/lib/golang.sh

```diff
@@ -294,6 +294,7 @@ kubeedge::golang::cross_build_place_binaries() {
     local name="${bin##*/}"
     if [ "${goarm}" == "8" ]; then
       set -x
+      GOARM="" # need to clear the value since golang compiler doesn't allow this env when building the binary for ARMv8.
       GOARCH=arm64 GOOS="linux" CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -ldflags "$ldflags" $bin
```

Comment:
we could specify `GOARM=""` just like set `GOARM=${goarm}` specifically for GOARM7 .
ref: https://github.com/golang/go/wiki/GoArm we should not set GOARM for GOARM8

Reply:
yes, that is why the command like this will fail, 
```make crossbuild WHAT=edgecore GOARM=GOARM8```

Reply:
make sense, in golang `GOARM=GOARM8` will failed.. I'm wondering whether it's reasonable ;-)